pyproject.toml

```diff
@@ -19,10 +19,15 @@ scipy = [
 ]
 pyarrow = "^16.0.0"
 pyyaml = "^6.0.1"
+fire = "^0.6.0"
+jinja2 = "^3.1.4"
```

Comment:
What is jinja going to be used for in our case? I am familiar with it for CellProfiler being a dependency.

Reply:
Thanks for checking on this - this I believe is needed for work found in #39 and inadvertently showed up here. I'll try removing it for this Pr to keep things clean.